addons/hr_calendar/models/res_partner.py

```diff
@@ -45,18 +45,18 @@ def _get_schedule(self, start_period, stop_period, everybody=False, merge=True):
             return {}
         interval_by_calendar = defaultdict()
         calendar_periods_by_employee = defaultdict(list)
-        employees_by_calendar = defaultdict(list)
+        resources_by_calendar = defaultdict(list)
 
         # Compute employee's calendars's period and order employee by his involved calendars
-        employees = sum(employees_by_partner.values(), start=self.env['hr.employee'])
-        calendar_periods_by_employee = employees._get_calendar_periods(start_period, stop_period)
+        resources = sum(employees_by_partner.values(), start=self.env['hr.employee'])
+        calendar_periods_by_employee = resources._get_calendar_periods(start_period, stop_period)
```

Comment:
Why rename here while the recordset is still an employee recordset?

Reply:
ah shit, my bad